src/components/List/List.stories.js

```diff
@@ -1,13 +1,42 @@
-/* @flow */
```

Comment:
If you did it due to the list of items, you can also make a small component `<Items />` that returns these.